### ISSUE TITLE
Repo is required in /etc/pacman.conf despite a dedicated pacman.conf is given to aur build

### ISSUE THREAD

**Issue by mipimipi** [AladW/aurutils#642]
I observed a strange phenomenon: I use `aur build` to build a package in a chroot container and add it to a repository. Via the `--pacman-conf` option, I provide the command with a dedicated `pacman.conf` file. Despite `aur build` is provided with this file, it requires the repository to be configured in `/etc/pacman.conf` in addition. Surprisingly, it is sufficient to only have a row `[myrepo]` in `/etc/pacman.conf` - without any other information (no `Server = ...` row). 

My scenario: I am using `aur build` to add packages to a remote repository. My script rsync's the remote repo to a local directory, `aur build` is called to add a package to local copy of the remote repo. Then the local copy is rsync'ed back to the remote repo and the local copy is deleted. By using a dedicated chroot container and a dedicated `pacman.conf` file, I would like to avoid that my system is affected by that package build/add process (i.e. the package shall not have to be installed to my local system and the repo shall not have to be configured in my `/etc/pacman.conf`).

The `aur build` statement:
```
aur build --database=<MYREPO> \
                  --arg-file=<LIST-OF-DIRS-WITH-PKGBUILDS> \
                  --root=<PATH-TO-LOCAL-REPO-DIRECTORY>" \ 
                  --chroot \
                  --directory=<PATH-TO-CHROOTDIR> \
                  --pacman-conf=<PATH-TO-DEDICATED-PACMAN.CONF> \
                  -- -c -u -T -- -c -i
```
The dedicated pacman.conf file contains a configuration for the local copy of the repo:

```
[myrepo]
SigLevel = Optional TrustAll
Server = file://<PATH-TO-LOCAL-REPO-DIRECTORY>
```
Why does the repo need to be configured in `/etc/pacman.conf`?

Thanks, mipi

**Comment by mipimipi** [comment]
I am pretty sure that I found the root cause of that behavior: In `aur chroot`, `pacman-conf` is called several times to query some config data. Consequently, it should be run against the custom pacman.conf (if the corresponding cli option has been set), but sometimes it is run against `/etc/pacman.conf` even if there is a custom pacman.conf. On my local machine, I changed the code of `aur chroot`, `pacman-conf` is now running against the custom pacman.conf (if it's existing). With this change my scenario works. I will submit a PR.
